data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApi.java

```diff
@@ -230,6 +231,9 @@ private void addValidatorHandlers(DataProvider dataProvider) {
         PostBlock.ROUTE,
         new PostBlock(validatorDataProvider, dataProvider.getSyncDataProvider(), jsonProvider));
     app.post(PostDuties.ROUTE, new PostDuties(validatorDataProvider, jsonProvider));
+    app.post(
+        PostAggregateAndProof.ROUTE,
+        new PostAggregateAndProof(validatorDataProvider, jsonProvider));
```

Comment:
generally this would mean we have a test case in `BeaconRestApiTest` also

Reply:
We do! :D

https://github.com/PegaSysEng/teku/pull/2518/files#diff-2e867c244b70539fa877a970c3b802dbR193-R197

Reply:
ok then :)